packages/framework/src/client.test.ts

```diff
@@ -1350,7 +1350,15 @@ describe('Novu Client', () => {
         workflowId: 'test-workflow',
         stepId: 'active-step-id',
         subscriber: {},
-        state: [],
+        state: [
+          {
+            stepId: 'skipped-step-id',
+            outputs: {},
+            state: {
+              status: 'success',
+            },
+          },
+        ],
```

Comment:
An empty `outputs` should have been supplied for this test, ensuring that a hydration can still occur for skipped steps. This fixes the assertion.